aptos-move/framework/aptos-framework/sources/transaction_validation.move

```diff
@@ -79,6 +79,30 @@ module aptos_framework::transaction_validation {
         txn_max_gas_units: u64,
         txn_expiration_time: u64,
         chain_id: u8,
+    ) {
+        prologue_common_v2(
```

Comment:
The function is private, we can add more arguments to it while preserving compatibility?